src/models/data-collection-class.js

```diff
@@ -0,0 +1,32 @@
+
+'use strict';
+
+class OverClass {
+    constructor(Smodel){
+        this.model = Smodel;
+        }
+
+        create(obj){
+            let newBook = this.model(obj)
+            return newBook.save();
```

Comment:
If you want you can leave how it is or you replace `newBook` with `newRecord` to make it more generic